tamr_unify_client/models/attribute/resource.py

```diff
@@ -0,0 +1,48 @@
+from tamr_unify_client.models.attribute.type import AttributeType
+from tamr_unify_client.models.base_resource import BaseResource
+
+
+class Attribute(BaseResource):
+    """
+    A Unify Attribute.
+
+    See https://docs.tamr.com/reference#attribute-types
+    """
+
+    @classmethod
+    def from_json(cls, client, data, api_path):
+        return super().from_data(client, data, api_path)
+
+    @property
+    def relative_id(self):
+        return self.api_path
```

Comment:
The contract around `api_path` is unclear, so it's unclear whether this equivalence between `api_path` and `relative_id` is valid.

Reply:
`api_path` is `alias` when it exists, and `relative_id` when it does not. this distinction is useful for things like refreshing a unified dataset, where using the `relative_id` would hit `/datasets/{id}:refresh` rather than `/projects/{id}/unifiedDataset:refresh`. Agree that it's confusing. There's an existing RFC for suggestions to improve this #64

Reply:
I think that is temporarily true. Not sure this will last forever though. Might be worth an in-code `# NOTE: For now api_path and relative_id are the same, but this may change in the future.` to help someone fixing a future bug.

Reply:
Done.